app/assets/javascripts/pretty-text/addon/sanitizer.js

```diff
@@ -143,6 +143,11 @@ export function sanitize(text, allowLister) {
         }
       }
     },
+    onTag(tag, html, options) {
+      if (tag === 'video' && html.match(/autoplay/) && !html.match(/muted/)) {
```

Comment:
Hmmm... what happens if the video src includes the words 'autoplay' or 'muted' 🤔 Ideally we would do some proper HTML parsing here, but I'm not sure we have the functions available to do it...